vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -37,35 +45,58 @@ $( document ).ready(function() {
 		$("#changeBgImgAlert").hide();
 		$("#bgImage").on("click", function(e){
 		    e.preventDefault();
-		    $("#arrow").remove();
-		    var icon = $('<span id="arrow" data-feather="navigation-2" class="flex"></span>');
+		    $("#icon").remove();
+		    var icon = $('<span id="icon" data-feather="navigation-2" class="flex"></span>');
 		    icon.css('position', 'absolute');
 		    
 		    var posX = $(this).position().left;
-                    var posY = $(this).position().top;
+		    var posY = $(this).position().top;
 		    
 		    storeX = e.pageX - $(this).offset().left;
 		    storeY = e.pageY - $(this).offset().top;
 		    icon.css('left', storeX + posX);
 		    icon.css('top', storeY + posY);
 		    icon.css('color', 'red');
-		    icon.css('font-size', "15px");
+		    icon.css('font-size', "10px");
 		    
 		    $("#space").append(icon);
-		    feather.replace();
+		    feather.replace();	        
 		});
 		$("#createSpaceLinkAlert").show();
 	});
 	
+	$('#type').change(function() {
+		$("#icon").remove();
+		$("#label-visibility").remove();
+		var spaceLink = {};
+		spaceLink["x"] = storeX;
+		spaceLink["y"] = storeY;
+		spaceLink["spaceLinkLabel"] = $("#spaceLinkLabel").val();
+		spaceLink["type"] = $("#type").val();	
+		makeItVisible(spaceLink);
+	});	
+	
+	$('#spaceLinkLabel').change(function() {
+		$("#label-visibility").remove();
+		var spaceLink = {};
+		spaceLink["x"] = storeX;
+		spaceLink["y"] = storeY;
+		spaceLink["spaceLinkLabel"] = $("#spaceLinkLabel").val();
+		spaceLink["type"] = $("#type").val();		
+		makeItVisible(spaceLink);
```

Comment:
those two change functions seem to be mostly the same, so the the common code can be put into a different method to be called.